app/Helpers/database/player-game.php

```diff
@@ -313,11 +313,11 @@ function getUsersCompletedGamesAndMax(string $user): array
         LEFT JOIN GameData AS gd ON gd.ID = pg.game_id
         LEFT JOIN Console AS c ON c.ID = gd.ConsoleID
         LEFT JOIN UserAccounts ua ON ua.ID = pg.user_id
-        WHERE ua.User = :user
+        WHERE (ua.User = :user OR ua.display_name = :user2)
         AND gd.achievements_published > $minAchievementsForCompletion
         ORDER BY PctWon DESC, PctWonHC DESC, MaxPossible DESC, gd.Title";
 
-    return legacyDbFetchAll($query, ['user' => $user])->toArray();
+    return legacyDbFetchAll($query, ['user' => $user, 'user2' => $user])->toArray();
 }
 
 function getGameRecentPlayers(int $gameID, int $maximum_results = 10): array
```

Comment:
Progress links in the Recent Players list use the User value.

Reply:
Fixed in latest.